resources/assets/js/components/JobBuilderImpact/ImpactForm.tsx

```diff
@@ -0,0 +1,186 @@
+            <div>
+              <Field
+                name="teamImpact"
+                id="TeamImpact"
+                placeholder={intl.formatMessage(messages.teamPlaceholder)}
+                label={intl.formatMessage(messages.teamLabel)}
+                required
+                component={TextArea}
+              />
+            </div>
+            <span>
+              <FormattedMessage
+                id="formField.error"
+                defaultMessage="This input has an error."
+                description="Flag text for validation error"
+              />
+            </span>
+          </div>
+          <div data-c-grid-item="base(1of1)" data-c-input="textarea">
+            <p data-c-font-weight="bold" data-c-margin="bottom(normal)">
+              <FormattedMessage
+                id="jobBuilder.impact.hireHeader"
+                defaultMessage="How the new hire makes an impact:"
+                description="Header of Job Poster Builder Hire Impact Section"
+              />
+            </p>
+            <p data-c-margin="bottom(normal)">
+              <FormattedMessage
+                id="jobBuilder.impact.hireBody"
+                defaultMessage="Describe how the new hire will contribute in this role. Focus on the
+              value they’ll bring, not on specific tasks (you’ll provide these
+              later on). For example “In this role, you’ll contribute to…” or, “As
+              a member of this team, you’ll be responsible for helping us…”"
+                description="Body of Job Poster Builder Hire Impact Section"
+              />
+            </p>
+
+            <span>
+              <FormattedMessage
+                id="formField.required"
+                defaultMessage="Required"
+                description="Flag text for empty required field"
+              />
+            </span>
+            <div>
+              <Field
+                id="HireImpact"
+                name="hireImpact"
+                label={intl.formatMessage(messages.hireLabel)}
+                placeholder={intl.formatMessage(messages.hirePlaceholder)}
+                required
+                component={TextArea}
+              />
+            </div>
+            <span>
+              <FormattedMessage
+                id="formField.error"
+                defaultMessage="This input has an error."
+                description="Flag text for validation error"
+              />
+            </span>
+          </div>
+          <div data-c-alignment="centre" data-c-grid-item="base(1of1)">
+            {/* <!-- Modal trigger, same as last step. --> */}
+            <button
+              data-c-button="solid(c1)"
+              data-c-dialog-action="open"
+              data-c-dialog-id="example-dialog-01"
+              data-c-radius="rounded"
+              disabled={isSubmitting}
+              form="form"
+              type="submit"
+            >
+              <FormattedMessage
+                id="button.next"
```

Comment:
A given formattedMessage id should only ever exist in one place. "button.next" will probably be used somewhere else, so we should either change this id to something page-specific (eg jobBuilder.impact.submitButton) or extract it to a common file where it can be reused elsewhere.

Reply:
The `button.next` translation id is a common translation that could be used in other places. I don't think we have to extract out the whole formatted message so it is only used once.